url-dsl/src/test/scala/urldsl/language/PathWithQueryParametersSpec.scala

```diff
@@ -23,11 +27,17 @@ class PathWithQueryParametersSpec extends AnyFlatSpec with Matchers {
     ) should be(Right(UrlMatching((2019, "january"), (10, List("Orange juice", "Water")))))
 
     pathWithParams.matchRawUrl(
-      "http://localhost:8080/hello/2019/january?age=10&drinks=orange%20juice&drinks=water"
+      sampleUrl
     ) should be(
       Right(UrlMatching((2019, "january"), (10, List("orange juice", "water"))))
     )
 
+    (pathWithParams & param[(Int, Int)]("tuple")).matchRawUrl(
+      sampleUrl
+    ) should be(
+      Right(UrlMatching((2019, "january"), (10, List("orange juice", "water"), 44, 55)))
```

Comment:
As an example, with the old Tupler, this evaluated to the following:

UrlMatching((2019, "january"), ((10, List("orange juice", "water")), (44, 55)))

instead of the new and flatter:

UrlMatching((2019, "january"), (10, List("orange juice", "water"), 44, 55))

I think the new behaviour is good? Or at least, not bad.

Reply:
I think it is much better indeed. In the example above, if someone thinks that `(10, List("orange juice", "water"))` has meaning in itself, they should map it to a case class before tupling :+1: